grafana/dashboards/timeline.json

```diff
@@ -0,0 +1,718 @@
+          "refId": "A",
+          "select": [
+            [
+              {
+                "params": [
+                  "id"
+                ],
+                "type": "column"
+              }
+            ]
+          ],
+          "table": "candata",
+          "timeColumn": "datum",
+          "timeColumnType": "timestamp",
+          "where": [
+            {
+              "name": "$__timeFilter",
+              "params": [],
+              "type": "macro"
+            }
+          ]
+        }
+      ],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Timeline",
+      "transformations": [
+        {
+          "id": "organize",
+          "options": {
+            "excludeByName": {
+              "End": true,
+              "start_date_ts": false
+            },
+            "indexByName": {
+              "Action": 2,
+              "Duration": 7,
+              "End": 1,
+              "End Address": 4,
+              "SoC": 13,
+              "SoC Diff": 14,
+              "Start": 0,
+              "Start Address": 3,
+              "distance_km": 8,
+              "distance_mi": 9,
+              "end_date_ts": 20,
+              "end_path": 18,
+              "kWh": 12,
+              "odometer_km": 5,
+              "odometer_mi": 6,
+              "outside_temp_avg_c": 15,
+              "outside_temp_avg_f": 16,
+              "range_diff_km": 10,
+              "range_diff_mi": 11,
+              "start_date_ts": 19,
+              "start_path": 17
+            },
+            "renameByName": {
+              "action": "",
+              "end_address": "End",
+              "km_diff": "Km",
+              "kwh": "",
+              "minutediff": "Time",
+              "odometer": "",
+              "outside_temp_avg": "Temperature",
+              "rangediff": "Range Difference",
+              "soc": "",
+              "soc_diff": "SoC Difference",
+              "start_address": "Start",
+              "start_date": "Date",
+              "start_date_ts": ""
+            }
+          }
+        }
+      ],
+      "type": "table"
+    }
+  ],
+  "schemaVersion": 27,
+  "style": "dark",
+  "tags": [],
```

Comment:
If you add the `tesla` tag to the dashboard, it will be selectable in the drop-down list. You can also add the link to the TeslaMate page and the dashboard picker via the settings to be consistent with the other dashboards.

Reply:
Added the tag. If you could add the link to the TeslaMate page and the dashboard picker? I don't dare to.:-)